documentation/bibliography.bib

```diff
@@ -836,6 +992,31 @@ @book{ByrdFriedman1954
   title     = {Handbook of elliptic integrals for engineers and physicists},
 }
 
+@book{Capderou2012,
+  author    = {Capderou, M.},
+  publisher = {Springer-Verlag France},
+  date      = {2012},
+  isbn      = {978-2-287-99049-6},
+  title     = {Satellites: de Kepler au GPS},
```

Comment:
`Satellites~: de Kepler au GPS`.

Reply:
Also did the same for Kahan's mindless question mark.